client/flags.go

```diff
@@ -80,8 +80,8 @@ func PostCommands(cmds ...*cobra.Command) []*cobra.Command {
 		c.Flags().Uint64(FlagAccountNumber, 0, "AccountNumber number to sign the tx")
 		c.Flags().Uint64(FlagSequence, 0, "Sequence number to sign the tx")
 		c.Flags().String(FlagMemo, "", "Memo to send along with transaction")
-		c.Flags().String(FlagFees, "", "Fees to pay along with transaction; eg: 10stake,1atom")
```

Comment:
not a fan of replacing with `...` doesn't convey how this field is used if anything maybe replace with `10stake,7foobar`